bitnami/mongodb/templates/_helpers.tpl

```diff
@@ -15,6 +15,36 @@ If release name contains chart name it will be used as a full name.
 {{- include "common.names.fullname" . -}}
 {{- end -}}
 
+{{/*
+Create a default mongo service name which can be overridden.
+*/}}
+{{- define "mongodb.servicename" -}}
+    {{- if .Values.service -}}
+        {{- if .Values.service.name }}
```

Comment:
Change the template to use service.nameOverride instead servicename

Reply:
Fixed